.github/workflows/daily-build.yml

```diff
@@ -6,9 +6,6 @@ on:
   schedule:
     - cron: "6 0 * * *"
 
-permissions:
-  contents: read
```

Comment:
Does this action need write permissions? Can those be given on a per-job basis while leaving this line in?

Reply:
Maybe - It might be from debugging. I will try

Reply:
No - a deeper understanding of permissions will be needed.

Reply:
I'm guessing that if you leave in the global read permissions line here, you still need to set write permissions for each job that needs it in this file, even if the workflows for those jobs are setting that.